UnityProject/Assets/Scripts/Items/Others/HandTeleporter.cs

```diff
@@ -45,6 +48,22 @@ public bool WillInteract(HandActivate interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(HandActivate interaction)
 		{
+			var worldPosEntrance = objectPhysics.OfficialPosition.RoundToInt();
+
+			if (ignoreInhibitors == false)
+			{
+				foreach(TeleportInhibitor inhib in TeleportInhibitor.Inhibitors)
```

Comment:
might be worth doing a circle overlap, to see if there's any inhibitors , Though you need to set the Collider to acustom one?

Reply:
I did it this way is because each inhibitor will have its own range centered on itself.  Like an inhibitor might have a range of 200 but a second one might only have a range of 10, so a circle overlap wont pick that up unless it's has a ridicously large range, and it also requires like you mentioned, custom colliders. Maybe there's a third method? but I think doing simple range checks here is the best option

Reply:
ok cool